### PR TITLE
Add List type and IN and CONTAINS operators

### PR DESCRIPTION
**Pull request by stephen-byrne**
Add `LIST` type, and `in` and `contains` operators.

### CODE REVIEW

**Review by BenSmith-MFJ — APPROVED** [review]

**Comment by BenSmith-MFJ** [comment]
Just out of curiosity, what was the use case for the new type and operators?

**Comment by stephen-byrne** [comment]
> Just out of curiosity, what was the use case for the new type and operators?

I am replacing Codot's validation rules with expr expressions.

File: src/test/java/io/mfj/expr/ExprTest.kt [hunk 1]
```diff
@@ -9,7 +9,7 @@ You may obtain a copy of the License at
 
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KinD, either express or implied.
```
[the remark this answers is not in the document]
Comment (stephen-byrne) [reply src/test/java/io/mfj/expr/ExprTest.kt]: fixed in fcfb8d0

**Review by dkilmer — APPROVED** [review]
easy peasy

**Pull request merged**